### ISSUE TITLE
Bass is too emphasized in visualizers

### ISSUE THREAD

**Issue by gitihobo** [rocksdanister/audio-visualizer-wallpaper#16]
I want to see the higher frequencies more, anyway I can change this?

**Comment by TotallyNotJeff** [comment]
> I want to see the higher frequencies more, anyway I can change this?

You can add a multiplier to the ends of the following lines (there are two identical) in the viz.js file to increase the overall gain of the visualizer:

`midY - (audioArray[x] / maxVal) * max_height * 5`

add the multiplier to both lines. I went with five since that was the most noticeable

not entirely sure if this is what you wanted but hope it helps

my modified visualizer file if you want to skip this part
[test 5.zip](https://github.com/rocksdanister/audio-visualizer-wallpaper/files/12777758/test.5.zip)

**Comment by TlN3d** [comment]
What is this file's name in the Music TV wallpaper?

**Comment by TotallyNotJeff** [comment]
> What is this file's name in the Music TV wallpaper?

At first glance, it seems like the Music TV has a built in visualizer in the same **three.module.js** file with the rest of scene construction, instead of a separate vizualizer file.

Haven't tested myself so I would recommend asking GPT to analyze the file for you and which lines to modify.

TL;DR
Music.TV.HQ / js / threejs / three.module.js

No separate visualizer file.

**Comment by rocksdanister** [comment]
Nah, I just put everything on index.html

**Comment by TlN3d** [comment]
Thank you I found it in index.html
